### PR TITLE
[VEN-2817]: Deployment files for the Pool lens, IL and NTG on the OP mainnet

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves # VEN-2817

Deployed Addresses:
```
    "ComptrollerBeacon": "0x64f9306496ccF7b7369d02d68D6abcA2Edfb871d",
    "ComptrollerImpl": "0x4D3f690A33A365Fc131777ea6e0f5B8821eb755b",
    "Comptroller_Core": "0x5593FF68bE84C966821eEf5F0a988C285D5B7CeC",
    "DefaultProxyAdmin": "0xeaF9490cBEA6fF9bA1D23671C39a799CeD0DCED2",
    "NativeTokenGateway_vWETH_Core": "0x5B1b7465cfDE450e267b562792b434277434413c",
    "PoolLens": "0x142160A2E699e33af337741f157D96aAd6bC72aA",
    "PoolRegistry": "0x147780799840d541C1d7c998F0cbA996d11D62bb",
    "PoolRegistry_Implementation": "0x6a166fcd39BA9c4ACc1b98eC45Adcdc4926E7967",
    "VTokenBeacon": "0xd550Bdfa9402e215De0BabCb99F7294BE0268367",
    "VTokenImpl": "0x5794a3D0238E18AA6de78e9095fF6a9A188A128d",
    "VToken_vOP_Core": "0x6b846E3418455804C1920fA4CC7a31A51C659A2D",
    "VToken_vUSDC_Core": "0x1C9406ee95B7af55F005996947b19F91B6D55b15",
    "VToken_vUSDT_Core": "0x37ac9731B0B02df54975cd0c7240e0977a051721",
    "VToken_vWBTC_Core": "0x9EfdCfC2373f81D3DF24647B1c46e15268884c46",
    "VToken_vWETH_Core": "0x66d5AE25731Ce99D46770745385e662C8e0B4025"
```

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: helpers/deploymentConfig.ts [hunk 1]
```diff
@@ -5322,6 +5332,160 @@ export const globalConfig: NetworkConfig = {
...
     preconfiguredAddresses: preconfiguredAddresses.opsepolia,
   },
+  opmainnet: {
+    tokensConfig: [
+      {
+        isMock: false,
+        name: "USD Coin",
+        symbol: "USDC",
+        decimals: 6,
+        tokenAddress: "0x0b2c639c533813f4aa9d7837caf62653d097ff85",
```
Comment (GitGuru7) [inline helpers/deploymentConfig.ts:5342]: Why we have this address `0x7F5c764cBc14f9669B88837ca1490cCa17c31607` of USDC in doc?

File: helpers/deploymentConfig.ts [hunk 2]
```diff
@@ -5322,6 +5332,160 @@ export const globalConfig: NetworkConfig = {
...
+            baseRatePerYear: "0",
+            multiplierPerYear: convertToUnit("0.06875", 18),
+            jumpMultiplierPerYear: convertToUnit("2.5", 18),
+            kink_: convertToUnit("0.8", 18),
+            collateralFactor: convertToUnit("0.75", 18),
+            liquidationThreshold: convertToUnit("0.78", 18),
+            reserveFactor: convertToUnit("0.1", 18),
+            initialSupply: convertToUnit("5000", 18), // 5,000 USDC
+            supplyCap: convertToUnit("10000000", 18),
+            borrowCap: convertToUnit("9000000", 18),
```
Comment (GitGuru7) [inline helpers/deploymentConfig.ts:5475]:
```suggestion
            borrowCap: convertToUnit("9000000", 6),
```

File: helpers/deploymentConfig.ts [hunk 3]
```diff
@@ -5322,6 +5332,160 @@ export const globalConfig: NetworkConfig = {
...
+            rateModel: InterestRateModels.JumpRate.toString(),
+            baseRatePerYear: "0",
+            multiplierPerYear: convertToUnit("0.06875", 18),
+            jumpMultiplierPerYear: convertToUnit("2.5", 18),
+            kink_: convertToUnit("0.8", 18),
+            collateralFactor: convertToUnit("0.75", 18),
+            liquidationThreshold: convertToUnit("0.78", 18),
+            reserveFactor: convertToUnit("0.1", 18),
+            initialSupply: convertToUnit("5000", 18), // 5,000 USDC
+            supplyCap: convertToUnit("10000000", 18),
```
Comment (GitGuru7) [inline helpers/deploymentConfig.ts:5474]:
```suggestion
            supplyCap: convertToUnit("10000000", 6),
```

File: helpers/deploymentConfig.ts [hunk 4]
```diff
@@ -5322,6 +5332,160 @@ export const globalConfig: NetworkConfig = {
...
+            symbol: "vUSDT_Core",
+            rateModel: InterestRateModels.JumpRate.toString(),
+            baseRatePerYear: "0",
+            multiplierPerYear: convertToUnit("0.06875", 18),
+            jumpMultiplierPerYear: convertToUnit("2.5", 18),
+            kink_: convertToUnit("0.8", 18),
+            collateralFactor: convertToUnit("0.75", 18),
+            liquidationThreshold: convertToUnit("0.78", 18),
+            reserveFactor: convertToUnit("0.1", 18),
+            initialSupply: convertToUnit("4998.602725	", 6), // 4,998.602725 USDT
```
Comment (chechu) [inline helpers/deploymentConfig.ts:5437]:
```suggestion
            initialSupply: convertToUnit("4998.602725", 6), // 4,998.602725 USDT
```

File: helpers/deploymentConfig.ts [hunk 5]
```diff
@@ -5322,6 +5332,160 @@ export const globalConfig: NetworkConfig = {
...
+            symbol: "vUSDC_Core",
+            rateModel: InterestRateModels.JumpRate.toString(),
+            baseRatePerYear: "0",
+            multiplierPerYear: convertToUnit("0.06875", 18),
+            jumpMultiplierPerYear: convertToUnit("2.5", 18),
+            kink_: convertToUnit("0.8", 18),
+            collateralFactor: convertToUnit("0.75", 18),
+            liquidationThreshold: convertToUnit("0.78", 18),
+            reserveFactor: convertToUnit("0.1", 18),
+            initialSupply: convertToUnit("5000", 18), // 5,000 USDC
```
Comment (chechu) [inline helpers/deploymentConfig.ts:5473]:
```suggestion
            initialSupply: convertToUnit("5000", 6), // 5,000 USDC
```

File: helpers/deploymentConfig.ts (hunk 2 above)
Comment (chechu) [reply helpers/deploymentConfig.ts]: b83fbc322ab2093f3264ab06f7db730b17c31b15

File: helpers/deploymentConfig.ts (hunk 3 above)
Comment (chechu) [reply helpers/deploymentConfig.ts]: b83fbc322ab2093f3264ab06f7db730b17c31b15

File: helpers/deploymentConfig.ts (hunk 4 above)
Comment (chechu) [reply helpers/deploymentConfig.ts]: b83fbc322ab2093f3264ab06f7db730b17c31b15

File: helpers/deploymentConfig.ts (hunk 5 above)
Comment (chechu) [reply helpers/deploymentConfig.ts]: b83fbc322ab2093f3264ab06f7db730b17c31b15

**Review by chechu — APPROVED** [review]

**Review by GitGuru7 — DISMISSED** [review]

**Review by chechu — APPROVED** [review]

**Pull request merged**